app/collins/solr/SolrConfig.scala

```diff
@@ -16,17 +16,18 @@ object SolrConfig extends Configurable {
       messageWithDefault("invalidUrl", "solr.externalUrl %s is invalid".format(t), t)
   }
 
+  # defaults taken from http://wiki.apache.org/solr/Solrj#Changing_other_Connection_Settings
   def embeddedSolrHome = getString("embeddedSolrHome", "NONE")
-  def enabled = true
+  def enabled = getBoolean("enabled", false)
```

Comment:
The solr plugin can not ever be disabled, so forcing this to true is ok (collins no worky without solr)